cernopendata/modules/fixtures/data/records/cms-primary-datasets-Run2011A.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "abstract": {
-      "description": "<p>BTag primary dataset in AOD format from RunA of 2011. Run period from run number 160404 to 173692.</p> <p>This dataset contains all runs from 2011 RunA. The list of validated runs, which must be applied to all analyses, can be found in</p>",
```

Comment:
This PR amends the pp data abstracts similarly to HI data abstracts specified at https://github.com/cernopendata/opendata.cern.ch/issues/3381

FYI there remains some occurrences of non-standard phrase:

```console
• cms-primary-datasets.json
2468:      "description": "<p>BTau primary dataset in AOD format from RunA of 2010</p> <p>This dataset contains all runs directed to it from 2010 RunA. The selection to BTau primary dataset was separated from JetMETTau primary dataset starting from run n>
4193:      "description": "<p>JetMETTau primary dataset in AOD format from RunA of 2010</p> <p>This dataset contains all runs directed to it from 2010 RunA. The selection to JetMETTau primary dataset was separated to JetMET and BTau primary datasets star>
4436:      "description": "<p>JetMET primary dataset in AOD format from RunA of 2010</p> <p>This dataset contains all runs directed to it from 2010 RunA. The selection to JetMET primary dataset was separated from JetMETTau primary d
```

Plus some more RAW and AOD samples where a non-standard phrase is useful and where we'll be keeping them:

- cms-raw-datasets-Run2011A.json
- cms-raw-datasets-Run2010B.json
- cms-derived-nanoaod-outreach-DoubleMuParked.json
- cms-raw-datasets-Run2012B.json